src/platform/packages/shared/kbn-esql-validation-autocomplete/src/definitions/generated/operators.ts

```diff
@@ -381,6 +381,22 @@ const addDefinition: FunctionDefinition = {
   examples: [],
 };
 
+// Do not edit this manually... generated by scripts/generate_function_definitions.ts
+const castDefinition: FunctionDefinition = {
+  type: FunctionDefinitionTypes.OPERATOR,
+  name: '::',
+  description: i18n.translate('kbn-esql-validation-autocomplete.esql.definitions.cast', {
+    defaultMessage:
+      'The `::` operator provides a convenient alternative syntax to the TO_<type> [conversion functions](https://www.elastic.co/docs/reference/elasticsearch/query-languages/esql/esql-functions-operators#esql-type-conversion-functions).',
+  }),
+  preview: false,
+  alias: undefined,
+  signatures: [],
+  locationsAvailable: [Location.STATS],
```

Comment:
@drewdaemon this looks wrong here, no?

Reply:
The cast needs another location, we need to suggest it after a field so it doesnt follow the other operators logic